shared-ui/lib/typography.ts

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components';
```

Comment:
lgtm other than failing test, do we want to delete the duplicated version of this file / redirect to this shared version since we already created styled font types somewhere else in the repo before?

Reply:
nice catch!